### PR TITLE
Edit YFII strategies - in pool3 pool4 mefi can also vote

### PR DESCRIPTION
**Pull request by jin10086**

### CODE REVIEW

File: spaces/dfi/index.json [hunk 1]
```diff
@@ -9,5 +9,27 @@
...
   "min": 0,
-  "invalid": []
+  "invalid": [],
+  "strategies": [
+    ["erc20-balance-of", {
+      "address": "0xBa37B002AbaFDd8E89a1995dA52740bbC013D992",
+      "symbol": "YFII",
+      "decimals": 18
+    }],
+    ["erc20-balance-of", { //poo4
```
Comment (e-asia) [inline spaces/dfi/index.json:19]: JSON standard does not allow comments. Please remove them.
```suggestion
    ["erc20-balance-of", {
```

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/dfi/index.json (hunk 1 above)
Comment (jin10086) [reply spaces/dfi/index.json]: done

**Review by e-asia — APPROVED** [review]

File: spaces/dfi/index.json (hunk 1 above)
Comment (e-asia) [reply spaces/dfi/index.json]: The UI will break currently if there is more than 2 strategies, does yours strategies have all the same value?

Comment (jin10086) [reply spaces/dfi/index.json]: use proxy vote contract https://etherscan.io/address/0x266cA21f966D93F5580E20b55ef66C8320E4760E#code

**Review by e-asia — APPROVED** [review]

File: spaces/dfi/index.json (hunk 1 above)
Comment (e-asia) [reply spaces/dfi/index.json]: @jin10086 The old proposal will show new strategy result, do you valid it?

Comment (jin10086) [reply spaces/dfi/index.json]: @e-asia How to verify the prod? I didn't find the document

Comment (e-asia) [reply spaces/dfi/index.json]: Look the difference between live and new strategies.
image
image

Comment (jin10086) [reply spaces/dfi/index.json]: Thank you, the new strategy will not affect.

**Pull request merged**